server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java

```diff
@@ -97,24 +97,26 @@ private void executeTask(CeTask task) {
 
   private static Profiler startProfiler(CeTask task) {
     Profiler profiler = Profiler.create(LOG);
-    return profiler.startInfo("Execute task | project={} | type={} | id={} | submitter='{}'",
-      task.getComponentKey(), task.getType(), task.getUuid(), getSubmitterLoginForLog(task));
+    return profiler.startInfo("Execute task | project={} | type={} | id={}{}",
+      task.getComponentKey(), task.getType(), task.getUuid(), submitterSuffix(task));
```

Comment:
yes, for parsing, time is the most relevant information, having it at the end make parsing easier